workflow/Snakefile

```diff
@@ -22,6 +22,14 @@
 
 import os
 import pandas as pd
+from snakemake.utils import validate
+
+###############################################################################
+### Configuration validation
+###############################################################################
+
+configfile: "config.yaml"
```

Comment:
But we don't want to hardcode the config file...

Btw: you can now remove all optional values from the config file (unless you wanna change the defaults) - so that should be part of this PR, too.